crates/sp-executor/src/lib.rs

```diff
@@ -59,6 +67,13 @@ impl sp_runtime::BoundToRuntimeAppPublic for ExecutorKey {
     type Public = ExecutorId;
 }
 
+/// Stake weight in the bundle election.
+pub type StakeWeight = u128;
+
+/// Unique identifier for each domain.
+// TODO: unify the DomainId usage across the codebase.
+pub type DomainId = u64;
```

Comment:
Ideal would to be to have all the primitives under primitves crates. Regarding the associate type, its just preference to keep type not very concrete on pallet side. Could be in a different PR for sure

Reply:
Yeah, it will be defined in the primitives crate for sure when the refactoring is done.

It's typical to use the associate type for the runtime specific type so that it's configurable for the runtime builder, but if it's a low level concept across the client and runtime, and no need to be configurable, using the concrete type is probably a better choice IMHO.